slik_wrangler/dqa.py

```diff
@@ -113,33 +120,43 @@ def check(col1, col2):
         log("No duplicate values in both rows and columns!!!", code='success')
 
 
-def consistent_structure_assessement(dataframe, display_findings=True):
+def structure_assessement(dataframe, display_findings=True):
     """
-    Checks the consitent nature of each feature column.
+    Checks the data structure of each feature column.
+    
+    Function checks if the data type of each feature column is consistent.
+    e.g. if there is an interger variable and a string variable within a
+    feature column or more.
     
-    It checks if the dtype across each feature column is consistent.
-    i.e. if there is an interger variable and a string variable across
-    the various feature columns.
+    For categorical columns. Assessment is conducted on similar categories
+    e.g. medium and Medium are the same and one have to be changed 
+    for the other, so it is either medium or Medium.
     
-    For categorical columns. assessment is made on duplicate categories
-    i.e. medium and Medium are the same categories and inconsistent.
+    This errors are mainly generated during data collection.
+    
+    Parameters:
+    ------------
+    dataframe: pandas Dataframe
+        This is the dataset to be evaluated for consistent data type.
+        
+    display_findings: boolean, Default True
+        Whether or not to display a dataframe highlighting
+        the feature columns with poor structures.
     """
     
     column_names = list(dataframe.columns)
     inconsistent_cols = []
     
     def check_consistent_in_type(_col):
         """
-        Checks for consistency in the dtype
+        Checks for consistency in the data type
```

Comment:
I think this is fine for sub-functions.
@Sensei-akin were you hoping for more descriptions here?

Reply:
@AdesholaAfolabi For some of the modules. The doc strings are not descriptive

Reply:
What models?

Reply:
Yeah it is!